solutions/unit-testing-service/src/app/user.service.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('When testing the UserService', () => {
       expect(http.get).toHaveBeenCalledWith(expectedUrl);
     });
 
-    it('Should get back an Observable of Users', async(() => {
+    it('Should get back an Observable of Users', fakeAsync(() => {
```

Comment:
There seem to be several things odd with this test case:

1.  const http is not used in this function
2.  it seems like fakeHttp returns fakeUserList in a synchronous manner with `Observable.of`.  Maybe fakeAsync is not needed at all for this to work?
3.  fakeAsync seem to need tick() to move the progression of time forward from the angular book : `https://angular-2-training-book.rangle.io/handout/testing/components/async.html`  That is not being called here.  Which is why I guessed this test is actually synchronous